src/get-started/beam-overview.md

```diff
@@ -36,6 +36,8 @@ Beam currently supports Runners that work with the following distributed process
          alt="Apache Flink">
 * Apache Spark <img src="{{ site.baseurl }}/images/logos/runners/spark.png"
          alt="Apache Spark">
+* Apache Gearpump(incubating) <img src="{{ site.baseurl }}/images/logos/runners/gearpump.png"
```

Comment:
Here there should be some caveat about it being under development and link to the "Ongoing projects" page.